assignments/function-conversion.js

```diff
@@ -1,30 +1,32 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction = () => {
+console.log("Function was invoked!");
+};
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = (param) => {
+  return param;
+};
```

Comment:
these all work and look good, but remember that a big feature of arrow functions is to get rid of brackets and return statements. this function can look like this:
```
let anotherFunction = (param) => param;

Reply:
ahh forgot that, thanks